pennylane_sphinx_theme/navbar.py

```diff
@@ -4,7 +4,7 @@
 
 NAVBAR_LEFT = [
     {
-        "name": "Quantum machine learning",
+        "name": "Learn",
         "href": "https://pennylane.ai/qml/",

```

Comment:
@Anakhom does the URL also need to be updated?

Reply:
I think no, it will take the user to `/qml `which is where the superhub currently sits. Also here are the [pathways](https://docs.google.com/document/d/12TTsdBTTBoJ9aYi2NgMnDct_5Sxa7k-K2Xo7fbUAjqY/edit) @BM7878 provided some time ago!

Reply:
ah of course 🤦‍♂️ I realized suddenly while reviewing the superhub!